lib/administrate/field/enum.rb

```diff
@@ -12,6 +12,16 @@ def html_options
         options[:html] || {}
       end
 
+      def evaluator
+        options[:evaluator] || Proc.new { true }
+      end
```

Comment:
after letting this name marinate for a day wdyt about calling this `option_filter`?